src/site/stages/build/drupal/menus.js

```diff
@@ -0,0 +1,332 @@
+/* eslint-disable no-param-reassign */
+
```

Comment:
I'd like to change this later so we don't have to do all this crazy manipulation, so I want to call out why there's so much complexity in here.
```suggestion

/**
 * Most of this file is to massage the data for the menu links
 * to match the structure in `vetsgov-content` so the mega
 * menu React component successfully renders all the links.
 */
 
```